dapr-spring/dapr-spring-boot-starters/dapr-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@artur-ciocanu do we need this?

Reply:
@artur-ciocanu  oh.. ok.. so maybe this is where we create the spring boot starters for Data and Messaging which shows how all comes together with the right dependencies. I think that here is also the place to have a combined starter for all Dapr submodules. This would be the place to have tests to demonstrate what the user should provide and what will be automatically wired by the spring boot autoconfiguration for the Dapr client.

Reply:
Yes exactly, we use starters to bootstrap auto-configurations. We will have one that bundles all the beans, then we can have one for `data`, one for `messaging`, etc.